docs/content/docs/usage/cms/netlify.md

```diff
@@ -32,14 +32,12 @@ with 2 files :
   <html>
     <head>
       <meta charset="utf-8" />
-      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
+      <meta name="viewport" content="width=device-width,  initial-scale=1.0" />
       <title>Content Manager</title>
-      <link rel="stylesheet" href="//cms.netlify.com/assets/cms.css" />
+      <link rel="stylesheet" href="https://unpkg.com/netlify-cms@^0.1.1/dist/cms.css" />
     </head>
     <body>
-      <script src="//cms.netlify.com/assets/vendor.js"></script>
-      <script src="//cms.netlify.com/assets/cms.js"></script>
+      <script src="https://unpkg.com/netlify-cms@^0.1.1/dist/cms.js"></script>
```

Comment:
Do we really need the ^0.1.1 in the url? I guess html won't change that much even if there is breaking change somewhere else?